api/app/src/routes/medical/link.ts

```diff
@@ -0,0 +1,96 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import { asyncHandler, getCxIdOrFail, getPatientIdFromQueryOrFail } from "../util";
+const router = Router();
+import status from "http-status";
+import { getPatient } from "../../command/medical/patient/get-patient";
+
+/** ---------------------------------------------------------------------------
+ * POST /link
+ *
+ * Creates link to the specified entity.
+ *
+ * @return  {Link}  The created link.
+ */
+router.post(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getCxIdOrFail(req);
+    const patientId = getPatientIdFromQueryOrFail(req);
+    const patient = await getPatient({ id: patientId, cxId });
+    // get linkSource from query params -> this should be "CommonWell"
+    console.log(patient);
+
+    // CommonWell.patientLink();
+    // store the following data in our DB in the patient:
+    //    - link ID
+    //    - person ID
+    // can put this into a JSONB column `link_data` with structure:
+    //    {["CommonWell"]: {cw_link_id, cw_person_id}}
+
+    return res.status(status.OK).json({});
+  })
+);
+
+/** ---------------------------------------------------------------------------
+ * DELETE /link
+ *
+ * Removes the specified HIE link from the specified patient.
+ *
+ * @return  200
+ */
+router.delete(
```

Comment:
Is this an actual delete meaning Lola 0? Or just revert it back to lola 1

Reply:
Based off this  _CommonWell.resetPatientLink_ were resetting so im assuming we are deleting from our db

Reply:
Yeah, this should be a reset in CW, but a delete from our DB. As per the SDK comment, we never actually want to delete the link in CW unless we get an explicit request from the patient, as we'll never be able to link to that person again.